src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/ia/CftQueryServiceAssignTaskTest.java

```diff
@@ -74,148 +67,6 @@ void setUp() {
         );
     }
 
-    @ParameterizedTest(name = "{0}")
-    @MethodSource("getGrantTypes")
-    void should_retrieve_a_task_to_assign(GrantType grantType) {
```

Comment:
Looks like we have removed this because it's IA centric. Does that mean we have the WA equivalent?

Reply:
Parameterized source provide only GrantType.STANDARD, GrantType.SPECIFIC. in wa/CftQueryServiceAssignTaskTest you already have two tests to cover those two grant type and some others.
should_retrieve_a_task_when_grant_type_standard
should_retrieve_a_task_when_grant_type_specific